forth-spec.el

```diff
@@ -0,0 +1,171 @@
+;;; forth-spec.el --- Browse words in Forth standard   -*-lexical-binding:t-*-
+;;
+;; Copyright (C) 2016  Helmut Eller <eller.helmut@gmail.com>
+;;
+;; This program is free software: you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;
+;; This package makes it convenient to browse documentation for
+;; standard Forth words from within Emacs.  The command
+;; `forth-spec-lookup' asks for the word name and invokes the HTML
+;; browser with the right URL.
+
+;;; Code:
+
+(eval-and-compile
+  (or (require 'cl-lib nil t)
+      ;; Emacs 23
+      (progn
+	(require 'cl)
+	(defmacro cl-ecase (&rest x) `(ecase . ,x)))))
```

Comment:
Well, cl-lib is the path of least resistence. This would be a non-issue, if we only had a module system ...